apps/demo/src/app/modules/angular-installation/angular-installation.template.html

```diff
@@ -8,14 +8,14 @@ <h2>Package installation</h2>
 
     <p>Install the core package if you do not have it yet:</p>
 
-    <code>npm i @microzord/core</code>
+    <code>npm i &#64;microzord/core</code>
```

Comment:
why?

Reply:
automigration from nx, i'll take a look

Reply:
error NG5002: Incomplete block "microzord". If you meant to write the @ character, you should use the "@" HTML entity instead.

Reply:
https://github.com/angular/angular/issues/52366

Reply:
Oh, sure
<img width="958" alt="image" src="https://github.com/taiga-family/microzord/assets/12021443/5894aec5-d09f-45f8-8e8a-ca3611ac6bf1">